hack/setup-cluster.sh

```diff
@@ -128,6 +128,11 @@ nodes:
 - role: control-plane
 EOF
 
+  # Disable ipvs on darwin arm64
+  if [[ "$OS" == "darwin" && "$ARCH" == "arm64" ]]; then
+    sed -i -E -e 's/kubeProxyMode: "ipvs"//g' "${config_file}"
+  fi
+
```

Comment:
What is the reason for this difference? Our E2E tests only work well with IPVS mode because Kubernetes is much slower in detecting network failures when using iptables.

Reply:
The Cluster networking would fail on my machine.
```
I1005 14:39:13.917657       1 node.go:141] Successfully retrieved node IP: 172.18.0.2
I1005 14:39:13.934021       1 server.go:632] "kube-proxy running in dual-stack mode" primary ipFamily="IPv4"
time="2023-10-05T14:39:13Z" level=warning msg="Running modprobe ip_vs failed with message: `modprobe: ERROR: could not insert 'ip_vs': Operation not permitted\ninsmod /lib/modules/5.15.0-84-generic/kernel/net/netfilter/ipvs/ip_vs.ko`, error: exit status 1"
time="2023-10-05T14:39:13Z" level=error msg="Could not get ipvs family information from the kernel. It is possible that ipvs is not enabled in your kernel. Native loadbalancing will not work until this is fixed."
I1005 14:39:13.942253       1 proxier.go:670] "Dummy VS not created" scheduler="rr"
E1005 14:39:13.942271       1 server.go:537] "Error running ProxyServer" err="can't use the IPVS proxier: Ipvs not supported"
E1005 14:39:13.942285       1 run.go:74] "command failed" err="can't use the IPVS proxier: Ipvs not supported"
Stream closed EOF for kube-system/kube-proxy-rlg7c (kube-proxy)
```

Reply:
I've had the same issue when setting up devcontainers. Maybe you need to do something similar to https://github.com/cloudnative-pg/cloudnative-pg/blob/main/.devcontainer/devcontainer.json#L21

Reply:
I think that's related to me using lima but Ill have a look 👍